crates/cli/commands/src/node.rs

```diff
@@ -107,6 +107,10 @@ pub struct NodeCommand<Ext: clap::Args + fmt::Debug = NoArgs> {
     /// Additional cli arguments
     #[command(flatten, next_help_heading = "Extension")]
     pub ext: Ext,
```

Comment:
I'd like to leverage this extension type for this (temp) feature

for example:

https://github.com/paradigmxyz/reth/blob/2bfa2defc4c8159d447905886b287046344465eb/bin/reth/src/optimism.rs#L25-L25